### DISCUSSION TITLE
processing whole folders with langchain

### DISCUSSION THREAD

**Discussion by pictoys** [logancyang/obsidian-copilot#146] [Ideas]
that would be a great feature

**Comment by logancyang** [comment]
Yes I have it in my plan. The quality of the QA can be a problem if the folder contains a lot of notes. I'm thinking if it's better to just be able to reference any note with the note title in the chat, it could be a better experience IMO. WDYT?

**Comment by pictoys** [comment]
I would have loved to include whole folders, but you're right it gets quite random with longer files. I really wonder why that is, since its original language model provides such a vast knowledge.
